tests/contrib/django/test_django_snapshots.py

```diff
@@ -25,15 +24,10 @@ def daphne_client(snapshot):
     # token) propagated to the new process.
     env = os.environ.copy()
     assert "_DD_TRACE_WRITER_ADDITIONAL_HEADERS" in env, "Client fixture needs test token in headers"
-    env.update(
-        {
-            "DJANGO_SETTINGS_MODULE": "tests.contrib.django.django_app.settings",
-        }
-    )
```

Comment:
why doesn't this need to be set anymore? Is this change related to the changes you have?

Reply:
I moved this to the asgi.py file. It makes more sense for the DJANGO_SETTINGS_MODULE to be defined there instead of setting it in our test

Reply:
if it doesn't have any impact on these changes, then let's remove it from this PR